oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/cache/NetworkAddressAliasEsDAO.java

```diff
@@ -62,9 +62,9 @@ public List<NetworkAddressAlias> loadLastUpdate(long timeBucketInMinute) {
 
             SearchResponse results =
                 getClient().search(NetworkAddressAlias.INDEX_NAME, search, params);
-            while (true) {
-                final String scrollId = results.getScrollId();
-                try {
+            final String scrollId = results.getScrollId();
+            try {
+                while (true) {
```

Comment:
This is wrong, `scrollId` is never updated and you'll be always scrolling to the same context (at line 82)

Reply:
![image](https://user-images.githubusercontent.com/3417650/170228950-ec842e00-f99c-480e-92e9-f3f96fd70608.png)

> This is wrong, `scrollId` is never updated.

Yes, the `scrollId` is never updated. So we can't delete the `scrollId` after executing the scroll once, so it cannot scroll it again based on the same `scrollId` for the second time.